packages/admin/admin/src/form/FieldContainer.tsx

```diff
@@ -13,6 +13,7 @@ export interface FieldContainerProps {
     warning?: string;
     scrollTo?: boolean;
     fieldMargin?: "always" | "never" | "onlyIfNotLast";
+    helperText?: string;
```

Comment:
Please use `React.ReactNode` here for better translatability (otherwise using `<FormattedMessage />` doesn't work.